package/MDAnalysis/lib/util.py

```diff
@@ -374,7 +375,9 @@ def _get_stream(filename, openfunction=open, mode='r'):
     """Return open stream if *filename* can be opened with *openfunction* or else ``None``."""
     try:
         stream = openfunction(filename, mode=mode)
-    except IOError:
+    except (IOError, OSError) as err:
```

Comment:
Aha I missed this change. So the error is getting raised here, when it should (apparently) be silencing the error here and raising it later (after `None` has been returned).  We shouldn't be changing how errors are propagated here